frontend/packages/console-app/src/components/volume-snapshot/create-volume-snapshot/create-volume-snapshot.tsx

```diff
@@ -256,7 +256,7 @@ const CreateSnapshotForm = withHandlePromise<SnapshotResourceProps>((props) => {
             </label>
             <SnapshotClassDropdown
               onChange={setSnapshotClassName}
-              id="claimName"
+              id="snapshotDropdown"
               selectedKey={snapshotClassName}
```

Comment:
This should be changed to `data-test=...` per our [best practices](https://github.com/openshift/console/blob/master/frontend/packages/integration-tests-cypress/README.md#best-practices).